billingportal_configuration.go

```diff
@@ -0,0 +1,218 @@
+	Products              []*BillingPortalConfigurationFeaturesSubscriptionUpdateProductParams `form:"products"`
+	ProrationBehavior     *string                                                              `form:"proration_behavior"`
+}
+
+// BillingPortalConfigurationFeaturesParams TODO docstring
+type BillingPortalConfigurationFeaturesParams struct {
+	CustomerUpdate      *BillingPortalConfigurationFeaturesCustomerUpdateParams      `form:"customer_update"`
+	InvoiceHistory      *BillingPortalConfigurationFeaturesInvoiceHistoryParams      `form:"invoice_history"`
+	PaymentMethodUpdate *BillingPortalConfigurationFeaturesPaymentMethodUpdateParams `form:"payment_method_update"`
+	SubscriptionCancel  *BillingPortalConfigurationFeaturesSubscriptionCancelParams  `form:"subscription_cancel"`
+	SubscriptionUpdate  *BillingPortalConfigurationFeaturesSubscriptionUpdateParams  `form:"subscription_update"`
+}
+
+// BillingPortalConfigurationParams TODO docstring
+type BillingPortalConfigurationParams struct {
+	Params           `form:"*"`
+	Active           *bool                                            `form:"active"`
+	BusinessProfile  *BillingPortalConfigurationBusinessProfileParams `form:"business_profile"`
+	DefaultReturnURL *string                                          `form:"default_return_url"`
+	Features         *BillingPortalConfigurationFeaturesParams        `form:"features"`
+}
+
+// BillingPortalConfiguration TODO docstring
+type BillingPortalConfiguration struct {
+	APIResource
+	Active           bool                                       `json:"active"`
+	Application      string                                     `json:"application"`
+	BusinessProfile  *BillingPortalConfigurationBusinessProfile `json:"business_profile"`
+	Created          int64                                      `json:"created"`
+	DefaultReturnURL string                                     `json:"default_return_url"`
+	Features         *BillingPortalConfigurationFeatures        `json:"features"`
+	ID               string                                     `json:"id"`
+	IsDefault        bool                                       `json:"is_default"`
+	Livemode         bool                                       `json:"livemode"`
+	Object           string                                     `json:"object"`
+	Updated          int64                                      `json:"updated"`
+}
+
+// BillingPortalConfigurationBusinessProfile TODO docstring
+type BillingPortalConfigurationBusinessProfile struct {
+	Headline          string `json:"headline"`
+	PrivacyPolicyURL  string `json:"privacy_policy_url"`
+	TermsOfServiceURL string `json:"terms_of_service_url"`
+}
+
+// BillingPortalConfigurationFeaturesCustomerUpdate TODO docstring
+type BillingPortalConfigurationFeaturesCustomerUpdate struct {
+	AllowedUpdates []*BillingPortalConfigurationFeaturesCustomerUpdateAllowedUpdate `json:"allowed_updates"`
```

Comment:
this should be `[]BillingPortalConfigurationFeaturesCustomerUpdateAllowedUpdate` since it's an array of strings (or here enums/consts)